### PR TITLE
Update documentation to reflect changes related to contexts

### PR DESCRIPTION
**Pull request by hygt**

### CODE REVIEW

**Comment by hygt** [comment]
Jenkins retest this please

File: docs/src/main/paradox/assets/api-reference/contexts/context-list.json [hunk 1]
```diff
@@ -1,23 +1,15 @@
...
-  },
+  "@context": "https://nexus.example.com/v0/contexts/nexus/core/resource",
   "total": 1,
   "results": [
     {
+      "@context": "https://nexus.example.com/v0/contexts/nexus/core/search",
       "resultId": "https://nexus.example.com/v0/contexts/myorg/mydom/mycontext/1.0.0",
       "source": {
         "@id": "https://nexus.example.com/v0/contexts/myorg/mydom/mycontext/1.0.0",
         "links": [
```
Comment (umbreak) [inline docs/src/main/paradox/assets/api-reference/contexts/context-list.json]: Links is now JSON-LD object, not a JSONAPI array
```json
"links": {
  "@context": "https://nexus.example.com/v0/contexts/nexus/core/search",
  "self": "https://nexus.example.com/v0/contexts/myorg/mydom/mycontext/1.0.0"
}
```

File: docs/src/main/paradox/assets/api-reference/contexts/context-list.json [hunk 2]
```diff
@@ -27,6 +19,7 @@
   ],
   "links": [
```
Comment (umbreak) [inline docs/src/main/paradox/assets/api-reference/contexts/context-list.json]: Same here, change links array for Json object

File: docs/src/main/paradox/assets/api-reference/domains/domain-list.json [hunk 3]
```diff
@@ -1,23 +1,15 @@
...
-  },
+  "@context": "https://nexus.example.com/v0/contexts/nexus/core/resource",
   "total": 1,
   "results": [
     {
+      "@context": "https://nexus.example.com/v0/contexts/nexus/core/search",
       "resultId": "https://nexus.example.com/v0/domains/myorg/mydom",
       "source": {
         "@id": "https://nexus.example.com/v0/domains/myorg/mydom",
         "links": [
```
Comment (umbreak) [inline docs/src/main/paradox/assets/api-reference/domains/domain-list.json]: Same here

File: docs/src/main/paradox/assets/api-reference/domains/domain-list.json [hunk 4]
```diff
@@ -27,6 +19,7 @@
   ],
   "links": [
```
Comment (umbreak) [inline docs/src/main/paradox/assets/api-reference/domains/domain-list.json]: Same here

File: docs/src/main/paradox/assets/api-reference/domains/existing-domain.json [hunk 5]
```diff
@@ -1,8 +1,10 @@
 {
-  "@context": {
-    "nxv": "https://nexus.example.com/v0/voc/nexus/core/",
-    "schema": "http://schema.org/"
-  },
+  "@context": [
```
Comment (umbreak) [inline docs/src/main/paradox/assets/api-reference/domains/existing-domain.json]: th order should be opposite right, first currently defined context and afterwards the resource context link. I think it is wrong order everywhere else too.

File: docs/src/main/paradox/assets/api-reference/instance-links.json [hunk 6]
```diff
@@ -1,14 +1,17 @@
 {
   "links": [
```
Comment (umbreak) [inline docs/src/main/paradox/assets/api-reference/instance-links.json]: Same here, change Json arr to Json object

File: docs/src/main/paradox/assets/api-reference/instance-search-list.json [hunk 7]
```diff
@@ -1,31 +1,22 @@
...
   "total": 102,
   "maxScore": 1,
   "results": [
     {
+      "@context": "https://nexus.example.com/v0/contexts/nexus/core/search",
       "resultId": "https://nexus.example.com/v0/data/myorg/mydom/2ndschema/v0.1.0/cb806141-b30d-4378-a37b-8bed354dce97",
       "score": 1,
       "source": {
         "@id": "https://nexus.example.com/v0/data/myorg/mydom/2ndschema/v0.1.0/cb806141-b30d-4378-a37b-8bed354dce97",
         "links": [
```
Comment (umbreak) [inline docs/src/main/paradox/assets/api-reference/instance-search-list.json]: Change to json object

File: docs/src/main/paradox/assets/api-reference/instances/instance-list.json [hunk 8]
```diff
@@ -31,6 +24,7 @@
   ],
   "links": [
```
Comment (umbreak) [inline docs/src/main/paradox/assets/api-reference/instances/instance-list.json]: change to Json object

File: docs/src/main/paradox/assets/api-reference/organizations/existing-organization.json [hunk 9]
```diff
@@ -1,8 +1,10 @@
 {
-  "@context": {
-    "nxv": "https://nexus.example.com/v0/voc/nexus/core/",
-    "schema": "http://schema.org/"
-  },
+  "@context": [
```
Comment (umbreak) [inline docs/src/main/paradox/assets/api-reference/organizations/existing-organization.json]: Change order

File: docs/src/main/paradox/assets/api-reference/pagination-links.json [hunk 10]
```diff
@@ -1,14 +1,17 @@
 {
   "links": [
```
Comment (umbreak) [inline docs/src/main/paradox/assets/api-reference/pagination-links.json]: Change to Jsonobject

**Pull request merged**